packages/kbn-grid-layout/grid/grid_panel.tsx

```diff
@@ -190,6 +190,7 @@ export const GridPanel = forwardRef<
               border-bottom: 2px solid ${euiThemeVars.euiColorSuccess};
               border-right: 2px solid ${euiThemeVars.euiColorSuccess};
               :hover {
+                opacity: 1;
```

Comment:
This was accidentally deleted in a previous PR 🤦 The resize handler should now be visible on hover

![Nov-01-2024 15-49-33](https://github.com/user-attachments/assets/36d6ba6c-61ce-430d-8a21-d96325ea2831)